ts_scripts/install_dependencies.py

```diff
@@ -1,12 +1,20 @@
 import os
 import platform
 import argparse
+import sys
+from pathlib import Path
+
+
+REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
+sys.path.append(REPO_ROOT)
+
+from ts_scripts.utils import check_python_version, is_gpu_instance
 
 
 class Common():
     def __init__(self):
         # Assumption is nvidia-smi is installed on systems with gpu
-        self.is_gpu_instance = True if os.system("nvidia-smi") == 0 else False
+        self.is_gpu_instance = is_gpu_instance()
         self.torch_stable_url = "https://download.pytorch.org/whl/torch_stable.html"
```

Comment:
@harshbafna @maaquib Please note that torchstable version is 1.7.0 at present. Let's verify this is not causing any issues if we are continuing to test on 1.6 till the other PR for 1.7 is merged.